ios/Capacitor/Capacitor/WebViewAssetHandler.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 import MobileCoreServices
 
-internal class CAPAssetHandler: NSObject, WKURLSchemeHandler {
+@objc(CAPWebViewAssetHandler)
+internal class WebViewAssetHandler: NSObject, WKURLSchemeHandler {
```

Comment:
Just curious, how come internal classes need an Obj-C name?

Reply:
Because inheriting from `NSObject`, which is necessary when conforming to certain protocols from UIKit, means that the class is bridged into the Obj-C runtime regardless of any access controls. Being internal, it's interface is not exposed outside of the framework but at runtime the class name would still exist in the flat namespace of the application.